index.html

```diff
@@ -61,6 +61,324 @@ <h1 class="display-5 fw-bold primary-text">I’m Yannick<br> Glad to see you!</h
+                 </ul>
+                 </div>
+                <div>
+              </div>
+              </div>
+              
+              <button class="btn btn-two" type="button">See Project</button>
+          </div>
+        </div>
+     
+     </div>
+  
+  </section>
+      
+      
+     <section class="work-section section-pd-50-50">
+
+     <div class="container">
+
+        <div class="flex-container">
+          <div class="flex-item-left work33">
+              
+            <h2 class="work-title">Uber Navigation</h2>
+              <br>
+              <br>
+              <div class="work-detail">
+                <div class="work-info">
+                 <ul>
+                   <li class="intro">UBER</li>
+                   <li> <i class="fa fa-circle"></i> Lead Developer</li>
+                   <li><i class="fa fa-circle"></i> 2018</li>
+                 </ul>
+                 </div>
+
+                 <p class="description">
+                    A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
+                 </p>
+
+                 <div class="work-tech">
+                 <ul>
+                   <li>html</li>
+                   <li>Ruby on Rails</li>
+                   <li>css</li>
+                   <li>javascript</li>
+                 </ul>
+                 </div>
+                <div>
+              </div>
+              </div>
+              
+              <button class="btn btn-one" type="button">See Project</button>
+            
+         </div>
+          
+          <div class="flex-item-right work3">
+              
+               <img src="image/portfolio4.png" class="img-responsive" alt="image-work1">
+ 
+          </div>
+        </div>
+     
+     </div>
+  
+  </section>
+      
+    
+  <section class="about-myself">
+        <div class="container py-5">
+            
+            <div class="row align-items-center justify-content-center">
+                                <div class="col-lg-6">
+
+                <h1 class="display-5 fw-bold primary-text">About Myself<br></h1>
+                 <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                 <a class="btn-connect">Let's connect</a>
+                <br><br>
+
+                <div class="media">
+                    <div class=""><a href="#"><i class="fa fa-twitter"></i></a></div>
+                    <div class=""><a href="#"><i class="fa fa-linkedin"></i></a></div>
+                    <div class=""><a href="#"><i class="fa fa-medium"></i></a></div>
+                    <div class=""><a href="#"><i class="fa fa-github"></i></a></div>
+                    <div class=""><a href="#"><i class="fa fa-angellist"></i></a></div>
+                </div>
```

Comment:
- [ ] In the About section, the list of icons should use a `ul` tag to group them, and `li` per icon.